php/libraries/Utility.class.inc

```diff
@@ -1348,5 +1348,27 @@ class Utility
             return $val;
         }
     }
+
+    /**
+     * Get the list of language available in the database
+     *
+     * @return array Array of language which exist in the database table 'language'
+     *               array is of the form
+     *               array($language_id => $language_label)
+     */
+    static function getLanguageList()
+    {
+        $DB =& \Database::singleton();
```

Comment:
This is going to cause a PHP warning about only variables being able to be assigned by reference

Reply:
solved